js_modules/dagster-ui/packages/ui-core/src/assets/AssetEvents.tsx

```diff
@@ -199,3 +229,71 @@ export const AssetEvents: React.FC<Props> = ({
     </>
   );
 };
+
+type EventType = 'observation' | 'materialization';
+
+const ALL_EVENT_TYPES: EventType[] = ['observation', 'materialization'];
+
+export const EventTypeSelect: React.FC<{
+  value: EventType[];
+  onChange: (value: EventType[]) => void;
+}> = ({value, onChange}) => {
```

Comment:
`React.FC` is no longer needed, since the component type no longer includes `children` implicitly. You can just apply the type directly to the props argument.